js/theme/fonts.js

```diff
@@ -51,11 +53,11 @@ export const makeFontStyleObject = (
 ): FontStyleObject => {
   if (os === 'android') {
     return {
-      fontFamily: makeFontFamilyName(os, weight, isItalic)
+      fontFamily: makeFontFamilyName(Platform.select, weight, isItalic)
```

Comment:
this method is obviously inconsistent now, since it both calls `Platform.select` AND gets the `os` string as input - `Platform.select` should be passed by the topmost scope, `makeFontStyleObject` should get `osSelect: PlatformSelectSpec<string, string> => string` as well